.pre-commit-config.yaml

```diff
@@ -25,3 +25,10 @@ repos:
     hooks:
       - id: nbstripout
         args: [--drop-empty-cells, --keep-output]
+
```

Comment:
Can we remove this comment block?

Reply:
Ideally if you stick to pyright the best way to be consistent would be to try would out how to put it into the pre-commit. However I didn't manage to do that here and so will remove the dead commented block.